BuildAssets/repos/publish-apt.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Required env vars:
+#export AWS_ACCESS_KEY_ID=$(get_octopusvariable "OctopusToolsAwsAccount.AccessKey")
+#export AWS_SECRET_ACCESS_KEY=$(get_octopusvariable "OctopusToolsAwsAccount.SecretKey")
+#export S3_PUBLISH_ENDPOINT=$(get_octopusvariable "Publish.APT.S3.TargetBucket")
+#export GPG_PRIVATEKEY=$(get_octopusvariable "Publish.APT.GPG.PrivateKey")
+#export GPG_PASSPHRASE=$(get_octopusvariable "Publish.APT.GPG.PassPhrase")
+
+echo "Importing private key"
+echo "$GPG_PRIVATEKEY" | gpg1 --batch --import 2>&1 || exit
+curl -s "https://s3.amazonaws.com/$S3_PUBLISH_ENDPOINT/public.key" | gpg1 --no-default-keyring --keyring trustedkeys.gpg --import 2>&1 || exit
+
+echo "Configuring S3 bucket"
+#aws s3 mb "s3://$S3_PUBLISH_ENDPOINT" || exit 1
```

Comment:
future improvement: let's figure out how to make this work between tentacle / octocli

Reply:
Addressed in #55.